app/src/main/java/org/gdg/frisbee/android/app/App.java

```diff
@@ -127,9 +131,46 @@ public void onCreate() {
         mLocationFinder = new GingerbreadLastLocationFinder(this);
         mLocationFinder.setChangedLocationListener(this);
         updateLastLocation();
+        
+        //Init TaggedEventSeries

```

Comment:
Please remove the comment and put the corresponding code in a `initTaggedEventSeries()` method.